src/device.c

```diff
@@ -1505,9 +1485,60 @@ void uvc_process_status_xfer(uvc_device_handle_t *devh, struct libusb_transfer *
                     event,
                     selector,
                     attribute,
-                    data, data_len,
+                    content, content_len,
```

Comment:
Or this may be a good time to do:

```
struct uvc_status_update {
  enum uvc_status_class status_class;
  int originator;
  int event;
  int selector;
  enum uvc_status_attribute status_attribute;
  void *data;
  size_t data_len;
};

typedef void(uvc_status_callback_t)(struct uvc_status_update, void* user_ptr);
```

or something like that.